src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -69,6 +69,7 @@ public class EditCommand extends Command {
 
     private final Nric nric;
```

Comment:
if you have a NricMatchesPredicate variable, I think having an Nric variable might no longer be necessary